### ISSUE TITLE
Lightning Navigator bar doesn't load in sandboxes

### ISSUE THREAD

**Issue by benji1304** [dannysummerlin/force-navigator#16]
I've reported this on the Chrome Store but figured it would be useful here too. The add-on works perfectly on my Production but not in either of my Sandboxes (Full and Dev) - none currently have a custom domain setup.

When pressing the shortcut key on my Mac I get this in the Chrome console
```
Uncaught TypeError: Cannot read property 'style' of null at <anonymous>:2:48
```
Which points to 
```
document.getElementById("sfnav_searchBox").style.zIndex = 9999
```
![image](https://user-images.githubusercontent.com/7213889/82161578-12044a00-98f2-11ea-8a92-def969441a4a.png)

**Comment by dannysummerlin** [comment]
@benji1304 thanks for bringing this up, sorry for the enormous delay! I have done some updates since this was posted, and I use it in sandboxes a good bit at this point, is it a) still not working for you and b) have you noticed any patterns with it, like particular pages in a sandbox, or just the whole thing?

**Comment by benji1304** [comment]
Hey @dannysummerlin ! 
I'll have another test and let you know. It's been a little while since I checked it properly 😉

**Comment by benji1304** [comment]
re-tested on my work laptop - absolutely fine now.

**Comment by CKloppel** [comment]
Hello @dannysummerlin 
Might need to reopen this.

This appears to be specific to some orgs.  Possibly a setting within Salesforce Setup.
![image](https://user-images.githubusercontent.com/50599251/124982657-31799d00-dff4-11eb-9b2d-7b6ccdd4305e.png)

In a personal developer org, and a very well lived in full copy sandbox, I'm able to use Navigator for Lightning without issue.  Just got access to a new Developer Sandbox and I'm seeing the same error listed above.

It is the entire sandbox, so not page dependent.

**Comment by dannysummerlin** [comment]
Hi @CKloppel , thanks for that info and screenshot, what the heck?! That would indicate that it never added the sfnav_searchBox element to the page - I could make it not error out by just doing a check for the box first, but if the box doesn't get added the extension won't be much help! Have you noticed any other errors, especially ones that have to do with XHR requests? I'll try to see where something might prevent sfnav_searchBox from being added to the page

**Comment by CKloppel** [comment]
I do see the following when refreshing the page.   Possibly related?

![image](https://user-images.githubusercontent.com/50599251/125139716-44f93680-e0ce-11eb-8673-6965f46cf619.png)

**Comment by benji1304** [comment]
I'm not seeing any issues in my sandboxes, tried full and dev.

**Comment by benji1304** [comment]
Weird-  i'm getting this issue again on my sandbox, but only in an incognito window! 

@dannysummerlin did we ever work out the cause or fix for this?

**Comment by dannysummerlin** [comment]
Hey Ben, sorry I have not had a chance to take a look at this in forever! If you wouldn’t mind grabbing some screenshots/captures of the newest errors and including them I’ll really try in the next few weeks to take a look. That’s is batty that it is only in incognito! (Sidenote another thing I am doing these days just to juggle logins is running Brave, Firefox, and Chrome separately, that way with incognito I can get up to six sessions going at once haha, don’t know if that sort of thing would be any help to you but may be a temporary way to address this).

**Comment by benji1304** [comment]
I can't seem to reproduce it again right now! I'll keep trying and update if possible

**Comment by dannysummerlinjstartorg** [comment]
Okay, I finally had a chance to do some cleanups and fixes, just pushed 2.8.1 to Chrome, will update Firefox soon. Thanks for all the details and back and forth folks!
